src/bilder/images/forum/deploy.py

```diff
@@ -56,6 +56,8 @@
     ),
 }
 
+files.directory("/opt/forum", present=True)
```

Comment:
I don't think the EC2 host needs to have the `/opt/forum` path, it just needs to be created in the Dockerfile for the Forum, and then you can set the `ENV DATA_DIR="/opt/forum"` in the dockerfile as well.